ogr/services/pagure/service.py

```diff
@@ -36,7 +37,7 @@ def __init__(
         instance_url: str = "https://src.fedoraproject.org",
         read_only: bool = False,
         insecure: bool = False,
-        max_retries: Union[int, requests.packages.urllib3.util.Retry] = 5,
+        max_retries: Union[int, urllib3.util.Retry] = 5,
```

Comment:
I've already fixed [here](https://github.com/packit/ogr/pull/673/files) and now realized it's not merged..;)